dotCMS/src/main/java/com/dotcms/rest/ClusterResource.java

```diff
@@ -49,13 +48,13 @@
      * @return
      * @throws DotStateException
      * @throws DotDataException
-     * @throws DotSecurityException
      * @throws JSONException
      */
     @GET
     @Path ("/getNodesStatus/{params:.*}")
     @Produces ("application/json")
-    public Response getNodesInfo ( @Context HttpServletRequest request, @PathParam ("params") String params ) throws DotStateException, DotDataException, DotSecurityException, JSONException {
+    public Response getNodesInfo ( @Context HttpServletRequest request, @PathParam ("params") String params )
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 46 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This method has 91 lines, which is greater than the 75 lines authorized. Split it into smaller methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS138)